packages/common/src/utils/nft.utils.ts

```diff
@@ -26,24 +23,30 @@ export class NftUtils {
   /**
    * Returns whether we should invoke our NFT handler for this tx
    * @param {Transaction} tx
+   * @param {string} network
```

Comment:
Missing `logger` parameter and wrong `network` type. Since it's a typescript file I think we could omit the types in the docstring.

Reply:
Fixed in 16efa8fdcf591e6e3c8b74a7e8e9e7bcb2f2d9e2